distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -68,6 +63,7 @@ org.codice.ddf.system.rootContext=/services
 #
 # System Information Properties
 #
+org.codice.ddf.system.branding=${branding-lowercase}
```

Comment:
I think you commented on this in a different pr but should this property name be more generic so it makes more sense for downstream projects?

Reply:
Yep, I did comment that, but I like @gordocanchola's idea to keep this property matching the convention of the other system properties, including `org.codice.ddf.system.siteName` which is project specific.